src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table-actions/table-actions.component.html

```diff
@@ -28,6 +28,7 @@
       <ng-container *ngFor="let action of dropDownActions">
         <button ngbDropdownItem
                 class="{{ toClassName(action['name']) }}"
+                [title]="useDisableDesc(action)"
```

Comment:
You should use this
```suggestion
                title="{{ useDisableDesc(action) }}"
```
otherwise a tooltip with the text `undefinded` will be displayed for buttons that do not have a description.
![screenshot](https://user-images.githubusercontent.com/1897962/85422164-91caa780-b575-11ea-92fb-1612b5730430.png)

Reply:
Fixed